docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -197,190 +281,152 @@ this might look something like:
-        , <&pro_micro 19 GPIO_ACTIVE_HIGH>  // col1 in the schematic
+        = <&pro_micro 10 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 11 GPIO_ACTIVE_HIGH>
+        , <&pro_micro 13 GPIO_ACTIVE_HIGH>
         ;
 };
-
 ```
 
-See the [Keyboard Scan configuration documentation](../../config/kscan.md) for details on configuring the KSCAN driver.
-
-### .conf files (Split Shields)
-
-While unibody boards only have one .conf file that applies configuration characteristics to the entire keyboard,
-split keyboards are unique in that they contain multiple .conf files with different scopes.
-For example, a split board called `my_awesome_split_board` would have the following files:
-
-- `my_awesome_split_board.conf` - Configuration elements affect both halves
-- `my_awesome_split_board_left.conf` - Configuration elements only affect left half
-- `my_awesome_split_board_right.conf` - Configuration elements only affect right half
+</TabItem>
+</SplitTabs>
 
-In most case you'll only need to use the .conf file that affects both halves of a split board. It's used for adding features like deep-sleep or rotary encoders.
+### Matrix Transform
 
-```ini title=my_awesome_split_board.conf
-CONFIG_ZMK_SLEEP=y
-```
+The matrix transform is used to transform row/column events into "key position" events.
 
-:::note
-The shared configuration in `my_awesome_split_board.conf` is only applied when you are building with a [`zmk-config` folder](../local-toolchain/build-flash.mdx#building-from-zmk-config-folder) and when it is present at `config/my_awesome_split_board.conf`. If you are not using a `zmk-config` folder, you will need to include the shared configuration in both `my_awesome_split_board_left.conf` and `my_awesome_split_board_right.conf` files.
-:::
-
-</TabItem>
-</Tabs>
+When a key is pressed, a kscan event is generated from it with a row value and a column value corresponding to the index (from 0) of the `row-gpios` pin and `col-gpios` pin respectively that triggered the event.
+The "key position" triggered is the index of the `RC(row, column)` in the matrix transform that correponds to the kscan event.
 
-## Matrix Transform
+<SplitTabs>
+<TabItem value="unibody">
+The `my_keyboard.overlay` must include a matrix transform that defines this mapping from row/column values to key positions.
+Add `#include <dt-bindings/zmk/matrix_transform.h>` to the top of the file.
 
-Internally ZMK translates all row/column events into "key position" events to maintain a consistent model that works no matter what any possible GPIO matrix may look like for a certain keyboard. This is particularly helpful when:
+Here is an example of a matrix transform for the previous 3x3 macropad:
 
-1. To reduce the used pins, an "efficient" number of rows/columns for the GPIO matrix is used, that does _not_ match the physical layout of rows/columns of the actual key switches.
-1. For non rectangular keyboards with thumb clusters, non `1u` locations, etc.
```

Comment:
While your new explanation seems good, it is pretty concise and I think it would be good to keep some of the explanations here, like these scenarios. When we have the dedicated explainer page we can remove some of the details.

Reply:
Added these back on the bottom of the page.

Reply:
Thanks, the new remarks look good.